src/constants/tokens.ts

```diff
@@ -512,6 +521,8 @@ export const USDT: { [chainId in ChainId]: Token } = {
     'USDT',
     'Tether USD',
   ),
+  // TODO(viet-nv): this address doesn't exist. check again
```

Comment:
todo

Reply:
this todo is fine. Let me refactor it later